sechub-cli/src/daimler.com/sechub/util/zip.go

```diff
@@ -121,31 +130,31 @@ func zipOneFolderRecursively(zipWriter *zip.Writer, folder string, zContext *zip
 		// Only accept source code files
 		isSourceCode := false
 		for _, srcPattern := range zContext.config.SourceCodePatterns {
-			if strings.HasSuffix(filePath, srcPattern) {
-				LogDebug(zContext.config.Debug, fmt.Sprintf("%q matches %q -> is source code", fileAbs, srcPattern))
+			if strings.HasSuffix(zipPath, srcPattern) {
+				LogDebug(zContext.config.Debug, fmt.Sprintf("%q matches %q -> is source code", file, srcPattern))
 				isSourceCode = true
 				break
 			}
 		}
 
 		// no matches above -> ignore file
 		if !isSourceCode {
-			LogDebug(zContext.config.Debug, fmt.Sprintf("%q no match with source code patterns -> skip", fileAbs))
+			LogDebug(zContext.config.Debug, fmt.Sprintf("%q no match with source code patterns -> skip", zipPath))
```

Comment:
The terms `source code patters` refer to file endings or how do we figure out if the file is a source code file?

Reply:
Yes, in the `for` loop above, the file name is checked if it has one of the suffixes, we regard as source code.
If nothing matches, then the file will not be added to the zip file.

Reply:
In this case it would make more sense to say: `%q not recognized as source code file as no file extension matched -> skip` or something like it.

Reply:
I have changed the text.